strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/repository/HostedRepositoryProvider.java

```diff
@@ -50,8 +53,9 @@ public ArtifactInputStream getInputStream(String storageId,
     {
         Repository repository = getConfiguration().getStorage(storageId).getRepository(repositoryId);
 
-        return getLayoutProviderRegistry().getProvider(repository.getLayout())
-                                          .getInputStream(storageId, repositoryId, path);
+        LayoutProvider layoutPtovider = getLayoutProviderRegistry().getProvider(repository.getLayout());
+        RepositoryPath repositoryPath = layoutPtovider.resolve(repository).resolve(path);
+        return (ArtifactInputStream) Files.newInputStream(repositoryPath);
```

Comment:
Cosmetic: (Here and in a number of places below and across the pull) Could you please leave an empty line before `return` statements? Thanks!